package.json

```diff
@@ -681,7 +681,7 @@
     "http-proxy": "^1.18.1",
     "i18n-iso-countries": "^4.3.1",
     "icalendar": "0.7.1",
-    "iedriver": "^3.14.2",
+
```

Comment:
Why is `iedriver` removed? How is Selenium driver related to this PR?

Reply:
ups, my mistake .... i had problems yarn kbn bootstraping ... need to revert this